packages/neuron-wallet/src/database/chain/ormconfig.ts

```diff
@@ -75,6 +75,7 @@ const connectOptions = async (genesisBlockHash: string): Promise<SqliteConnectio
       RemoveLiveCell1592781363749,
       AddIndexerTxHashCache1592727615004,
     ],
+    logger: 'simple-console',
```

Comment:
I am wondering what is this new config property `logger` for? Is it a new logging format in typeorm?

Reply:
Default options `advance-logger` parses and highlights sql query, which affects performance.
see: https://github.com/nervosnetwork/neuron/pull/1800/commits/e21e27b13d7bca8ad083bc912479486986306eff
and profile:
https://www.dropbox.com/s/m5ky04e7dl2ts5u/Heap-20200811T000034.heapsnapshot?dl=0
typeorm logger docs:
https://orkhan.gitbook.io/typeorm/docs/logging#changing-default-logger

Reply:
Got it. Is there a performance benchmark between these two log mode?

Reply:
> Got it. Is there a performance benchmark between these two log mode?

I just did a simple benchmark on my own, using a query that Neuron frequently warns about with `query-is-slow`:
https://github.com/yuche/typeorm-logger-bench

Benchmark shows that `advance-logger` is 20x-30x slower than `simple-logger`.
#### macos default Terminal:
simpleLogger x 38,057 ops/sec ±0.59% (94 runs sampled)
advanceLogger x 1,198 ops/sec ±1.14% (95 runs sampled)
#### iTerm 2
simpleLogger x 34,657 ops/sec ±6.23% (78 runs sampled)
advanceLogger x 1,033 ops/sec ±1.47% (84 runs sampled)

Reply:
Interesting to know 👍